site/profile/manifests/ts/hexrot.pp

```diff
@@ -0,0 +1,44 @@
+# @summary
+# Sets up repos and symlinks for hexrot
+
+class profile::ts::hexrot {
+  vcsrepo { '/opt/ts_config_mttcs':
```

Comment:
This is an important resource which needs spec coverage.

Reply:
yes, this was missing and was added just now